docs: How to add custom monitoring resources (#554)
kinvolk/lokomotive: docs/how-to-guides/add-new-dashboards-new-alerts.md

> # Add Custom Monitoring Configs

Reviewer: ```suggestion
# Add custom monitoring configs
```
and others

Author: Since this is a title, shouldn't it be capitalised?

Reviewer: We follow the `Sentence case`. https://github.com/kinvolk/style-guide/blob/master/style-guide.md#title-styling

## Contents

* [Introduction](#introduction)
* [Prerequisites](#prerequisites)
* [Add Custom Grafana Dashboards](#add-custom-grafana-dashboards)
* [Add new Service Monitors](#add-new-service-monitors)
  * [Default Prometheus Operator setting](#default-prometheus-operator-setting)
  * [Custom Prometheus Operator setting:](#custom-prometheus-operator-setting)
* [Add Custom Alerts for Alertmanager](#add-custom-alerts-for-alertmanager)
  * [Default Prometheus Operator setting](#default-prometheus-operator-setting-1)
  * [Custom Prometheus Operator setting:](#custom-prometheus-operator-setting-1)
* [Additional resources](#additional-resources)

## Introduction

This guide will take you through the steps of adding custom grafana dashboards custom alert manager rules.

## Prerequisites

- The component prometheus-operator deployed on a Lokomotive cluster.

## Add Custom Grafana Dashboards

> Create a Configmap with key as the dashboard file name and value as JSON dashboard, e.g.:

Reviewer: ```suggestion
Create a ConfigMap with key as the dashboard file name and value as JSON dashboard. Example:
```

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: dashboard
  labels:
    grafana_dashboard: "true"
data:
  metallb.json: |
    {
      "annotations": {
[REDACTED]
```

> Make sure that you add the label `grafana_dashboard: "true"` so that grafana automatically picks up the dashboards in the Configmaps all over the cluster.

Reviewer: ```suggestion
Make sure that you add the label `grafana_dashboard: "true"` so that Grafana automatically picks up the dashboards in the ConfigMaps all over the cluster.
```

Author: So what is the rule here, should we capitalise all occurrences of Grafana? How does it apply?

Reviewer: Grafana is the official name of the product like Kubernetes hence capital G , similar to `Kubernetes` and not `kubernetes`
